docs/appendices/release-notes/5.6.2.rst

```diff
@@ -69,3 +69,6 @@ Fixes
   A ``NULLABLE`` function in this context means a function returning ``NULL``
   if and only if the input is a ``NULL``.
 
+- Fixed an issue that caused exceptions with messages like
+  'ShardCollectContext already added' in low heap situations causing multiple
+  shards to be idle and be active simultaneously.
```

Comment:
I don't think this has anything to do with low heap, as the heap has no influence on whether shards are idle and active.

And I also don't think the amount of shards going from idle to active has any impact on _if_ the race condition happens. 


```suggestion
- Fixed a race condition that could lead to ``ShardCollectContext already
  added`` errors when making a query after a table had been idle without any
  accesses for a while.
```